### ISSUE TITLE
multigene display--unselect the gene cart

### ISSUE THREAD

**Issue by songeric1107** [IGS/gEAR#96]
add an option to unselect the gene cart.

If the user mis-select a gene cart, there is no option to unselect it.  Although user can chose "reset options", user has to reselect everything again.

![Screen Shot 2021-11-17 at 8 45 43 AM](https://user-images.githubusercontent.com/6730079/142212395-d9080d43-4650-45dd-a263-aa1a3f0b4463.png)

**Comment by adkinsrs** [comment]
You can just click the "X" to the far-right of the Gene Symbols dropdown and clear the genes

**Comment by songeric1107** [comment]
I refer to "load from gene cart", not the genes.

**Comment by adkinsrs** [comment]
Ok I will add a "clear" feature, but if you choose another gene cart entry it will actually overwrite the genes already the selected with the new gene cart

**Comment by songeric1107** [comment]
yes, that will be great for providing another option for "changing mind of using gene card"

**Comment by adkinsrs** [comment]
I have a commit in place for this.  Will push to gear-devel soon

**Comment by adkinsrs** [comment]
Pushed to gear-devel
